meta/runtime.yml

```diff
@@ -10,6 +10,10 @@ action_groups:
   - compute_resource
   - config_group
   - content_credential
+  - content_export_info
+  - content_export_library
+  - content_export_repository
+  - content_export_version
```

Comment:
```suggestion
```

These modules are not part of this PR, so they should not be in the metadata either ;)